src/MoeLens.sol

```diff
@@ -185,11 +192,12 @@ contract MoeLens {
 
     function getFarmDataAt(uint256 pid, address user) external view returns (Farm memory farm) {
         farm.pid = pid;
-        farm.isRewardable = _veMoe.isInTopPoolIds(pid);
-        farm.votesOnFarm = _veMoe.getVotes(pid);
```

Comment:
was this just a duplicate property?

Reply:
Yeah it's a duplicate of `totalVotesOnFarm` a few lines below